config.py

```diff
@@ -739,3 +735,6 @@ def create_transaction(db):
 
     # Feature Flag: Whether the repository action count worker is enabled.
     FEATURE_REPOSITORY_ACTION_COUNTER = True
+
+    # TEMP FEATURE: Backfill the sizes of manifests.
+    FEATURE_MANIFEST_SIZE_BACKFILL = True
```

Comment:
.... What the hell?

Its... gone....

I'll add it back in tomorrow

Reply:
Found it in the history and brought it back

Reply:
> TEMP FEATURE: Backfill the sizes of manifests.

Since this is mentioned as a temp feature, what is the condition to remove it? And, can you add that as a `TODO(after-something)` statement?

Reply:
It should be removed once it has been run on production and run on prem; so basically, we remove it in the next version